open_event/views/admin/models_views/event.py

```diff
@@ -301,6 +304,63 @@ def event_session_delete(self, event_id, session_id):
         return redirect(url_for('.event_sessions',
                                 event_id=event_id))
 
+    @expose('/<event_id>/session/<session_id>/review')
+    def session_reviews(self, event_id, session_id):
+        """Reviews for a Session"""
+        reviews = DataGetter.get_reviews(session_id)
+        self.name = 'Reviews'
+        return self.render('admin/model/review/list.html',
+                           objects=reviews,
+                           event_id=event_id,
+                           session_id=session_id)
+
+    @expose('/<event_id>/session/<session_id>/review/new', methods=('GET', 'POST'))
+    def session_review_new(self, event_id, session_id):
+        """Add Review for a Session"""
+        events = DataGetter.get_all_events()
+        form = ReviewForm()
+        self.name = 'Review'
+        if form.validate_on_submit():
+            is_created = DataManager.create_review(form, session_id)
+            if is_created:
+                flash('Thank you for your review')
+                return redirect(url_for('.session_reviews',
+                                event_id=event_id, session_id=session_id))
+            else:
+                flash('Sorry, review not added. It had invalid data.')
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.session_reviews',
+                               event_id=event_id, session_id=session_id))
+
+    @expose('/<event_id>/session/<session_id>/review/<review_id>/edit', methods=('GET', 'POST'))
+    def session_review_edit(self, event_id, session_id, review_id):
+        """Edit Review for a Session"""
+        review = DataGetter.get_object(Review, review_id)
+        events = DataGetter.get_all_events()
+        form = ReviewForm(obj=review)
+        self.name = 'Review {0} | Edit'.format(review_id)
+        if form.validate_on_submit():
+            DataManager.update_review(form, review)
+            flash("Review updated")
+            return redirect(url_for('.session_reviews',
+                                    event_id=event_id, session_id=session_id))
+        return self.render('admin/model/create_model.html',
+                           form=form,
+                           event_id=event_id,
+                           events=events,
+                           cancel_url=url_for('.session_reviews',
+                               event_id=event_id, session_id=session_id))
+
+    @expose('/<event_id>/session/<session_id>/review/<review_id>/delete', methods=('GET', 'POST'))
+    def session_review_delete(self, event_id, session_id, review_id):
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://www.codacy.com/app/triplez/open-event-orga-server/file/2406653970/issues/source?bid=3233864&fileBranchId=3233889#l358)